packages/plugin-commands-outdated/src/recursive.ts

```diff
@@ -153,6 +157,39 @@ ${renderCurrent(outdatedPkg)} ${chalk.grey('=>')} ${renderLatest(outdatedPkg)}`
     .join('\n\n') + '\n'
 }
 
+function renderOutdatedJSON (outdatedMap: Record<string, OutdatedInWorkspace>, opts: { long?: boolean }) {
```

Comment:
why isn't the similar function from outdated.ts reused here?

Reply:
I was wondering which or not I should re-implement `renderOutdatedJSON` also in `recursive.ts` or not. I decided to re-implement it because

- in recursive, `dependentPackages` field is added and this requires `OutdatedInWorkspace` (but this is trivial)
- `sortOutdatedPackages` function, which is used in both of `outdated` and recursive, is a bit different in `outdated` and recursive…